MANIFEST.in

```diff
@@ -1,4 +1,3 @@
-include *.py MANIFEST.in
-global-include *.txt *.rst *.ini *.yml *.cfg *.crt *.key
-global-exclude .gitignore
-prune build
+graft aiosmtpd
+include *LICENSE* *.cfg *.ini *.py *.rst *.yml
```

Comment:
Can you add `.coveragerc` here? It's the only important file missing.

Just `.coveragerc`. After running coverage there will be a file `.coverage` that should NOT be included.

Reply:
Sure thing!

Reply:
This is now included:

```
drwxr-xr-x dave/dave         0 2020-11-12 10:11 aiosmtpd-1.2.2/
-rw-r--r-- dave/dave       612 2020-11-12 10:05 aiosmtpd-1.2.2/.appveyor.yml
-rw-r--r-- dave/dave       365 2020-11-12 10:05 aiosmtpd-1.2.2/.coveragerc
-rw-r--r-- dave/dave       637 2020-11-12 10:05 aiosmtpd-1.2.2/.travis.yml
-rw-r--r-- dave/dave     11358 2020-11-12 10:05 aiosmtpd-1.2.2/LICENSE
-rw-r--r-- dave/dave        96 2020-11-12 10:05 aiosmtpd-1.2.2/MANIFEST.in
-rw-r--r-- dave/dave       686 2020-11-12 10:11 aiosmtpd-1.2.2/PKG-INFO
-rw-r--r-- dave/dave      6957 2020-11-12 10:05 aiosmtpd-1.2.2/README.rst
drwxr-xr-x dave/dave         0 2020-11-12 10:11 aiosmtpd-1.2.2/aiosmtpd/
-rw-r--r-- dave/dave         0 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/__init__.py
-rw-r--r-- dave/dave        71 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/__main__.py
-rw-r--r-- dave/dave      3153 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/controller.py
drwxr-xr-x dave/dave         0 2020-11-12 10:11 aiosmtpd-1.2.2/aiosmtpd/docs/
-rw-r--r-- dave/dave         7 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/docs/.gitignore
-rw-r--r-- dave/dave      6770 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/docs/Makefile
-rw-r--r-- dave/dave      7255 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/docs/NEWS.rst
-rw-r--r-- dave/dave         0 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/docs/__init__.py
-rw-r--r-- dave/dave      2768 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/docs/cli.rst
-rw-r--r-- dave/dave      4943 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/docs/concepts.rst
-rw-r--r-- dave/dave      9707 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/docs/controller.rst
-rw-r--r-- dave/dave     12936 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/docs/handlers.rst
-rw-r--r-- dave/dave      1934 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/docs/intro.rst
-rw-r--r-- dave/dave       630 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/docs/lmtp.rst
-rw-r--r-- dave/dave      1879 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/docs/manpage.rst
-rw-r--r-- dave/dave      2145 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/docs/migrating.rst
-rw-r--r-- dave/dave     11128 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/docs/smtp.rst
-rw-r--r-- dave/dave      7190 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/handlers.py
-rw-r--r-- dave/dave       601 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/lmtp.py
-rw-r--r-- dave/dave      5943 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/main.py
-rw-r--r-- dave/dave     37241 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/smtp.py
drwxr-xr-x dave/dave         0 2020-11-12 10:11 aiosmtpd-1.2.2/aiosmtpd/testing/
-rw-r--r-- dave/dave         0 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/testing/__init__.py
-rw-r--r-- dave/dave      2236 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/testing/helpers.py
drwxr-xr-x dave/dave         0 2020-11-12 10:11 aiosmtpd-1.2.2/aiosmtpd/tests/
-rw-r--r-- dave/dave         0 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/tests/__init__.py
drwxr-xr-x dave/dave         0 2020-11-12 10:11 aiosmtpd-1.2.2/aiosmtpd/tests/certs/
-rw-r--r-- dave/dave         0 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/tests/certs/__init__.py
-rw-r--r-- dave/dave      1415 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/tests/certs/server.crt
-rw-r--r-- dave/dave      1704 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/tests/certs/server.key
-rw-r--r-- dave/dave     25624 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/tests/test_handlers.py
-rw-r--r-- dave/dave      1829 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/tests/test_lmtp.py
-rw-r--r-- dave/dave     10899 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/tests/test_main.py
-rw-r--r-- dave/dave      1949 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/tests/test_server.py
-rw-r--r-- dave/dave     62926 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/tests/test_smtp.py
-rw-r--r-- dave/dave      2004 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/tests/test_smtps.py
-rw-r--r-- dave/dave      8151 2020-11-12 10:05 aiosmtpd-1.2.2/aiosmtpd/tests/test_starttls.py
drwxr-xr-x dave/dave         0 2020-11-12 10:11 aiosmtpd-1.2.2/aiosmtpd.egg-info/
-rw-r--r-- dave/dave       686 2020-11-12 10:11 aiosmtpd-1.2.2/aiosmtpd.egg-info/PKG-INFO
-rw-r--r-- dave/dave      1248 2020-11-12 10:11 aiosmtpd-1.2.2/aiosmtpd.egg-info/SOURCES.txt
-rw-r--r-- dave/dave         1 2020-11-12 10:11 aiosmtpd-1.2.2/aiosmtpd.egg-info/dependency_links.txt
-rw-r--r-- dave/dave        49 2020-11-12 10:11 aiosmtpd-1.2.2/aiosmtpd.egg-info/entry_points.txt
-rw-r--r-- dave/dave         9 2020-11-12 10:11 aiosmtpd-1.2.2/aiosmtpd.egg-info/requires.txt
-rw-r--r-- dave/dave        18 2020-11-12 10:11 aiosmtpd-1.2.2/aiosmtpd.egg-info/top_level.txt
-rw-r--r-- dave/dave      8923 2020-11-12 10:05 aiosmtpd-1.2.2/conf.py
drwxr-xr-x dave/dave         0 2020-11-12 10:11 aiosmtpd-1.2.2/examples/
-rw-r--r-- dave/dave         0 2020-11-12 10:05 aiosmtpd-1.2.2/examples/__init__.py
-rw-r--r-- dave/dave       196 2020-11-12 10:05 aiosmtpd-1.2.2/examples/client.py
-rw-r--r-- dave/dave       433 2020-11-12 10:05 aiosmtpd-1.2.2/examples/server.py
-rwxr-xr-x dave/dave      1150 2020-11-12 10:05 aiosmtpd-1.2.2/release.py
-rw-r--r-- dave/dave       136 2020-11-12 10:11 aiosmtpd-1.2.2/setup.cfg
-rw-r--r-- dave/dave      1114 2020-11-12 10:05 aiosmtpd-1.2.2/setup.py
-rw-r--r-- dave/dave      5095 2020-11-12 10:05 aiosmtpd-1.2.2/setup_helpers.py
-rw-r--r-- dave/dave      1727 2020-11-12 10:05 aiosmtpd-1.2.2/tox.ini
-rw-r--r-- dave/dave       361 2020-11-12 10:05 aiosmtpd-1.2.2/unittest.cfg
```